### PR TITLE
Add compat data for Body (Fetch API)

### PR DESCRIPTION
**Pull request by maboa**
I'm not sure how to represent the preferences when current data is given like so:

```
[1] Behind a preference in version 41.

[2] Behind a preference starting with version 34.

[3] Behind a preference in version 28.
```
(https://developer.mozilla.org/en-US/docs/Web/API/Body/bodyUsed)

The `flags.name` field is required and cannot be null. As a temporary measure I have left it as an empty string.

### CODE REVIEW

File: api/Body.json [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+          "firefox": [
+            {
+              "version_added": "39"
+            },
+            {
+              "version_added": "34",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": ""
```
Comment (Elchi3) [inline api/Body.json]: The name was "dom.fetch.enabled".

File: api/Body.json [hunk 2]
```diff
@@ -0,0 +1,593 @@
...
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "body": {
+        "__compat": {
+          "description": "The <code>body</code> read-only property",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/body",
```
Comment (Elchi3) [inline api/Body.json]:
```diff
-"https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/body"
+"https://developer.mozilla.org/docs/Web/API/Body/body"
```

File: api/Body.json [hunk 3]
```diff
@@ -0,0 +1,593 @@
...
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "body": {
+        "__compat": {
+          "description": "The <code>body</code> read-only property",
```
Comment (Elchi3) [inline api/Body.json]: I think the description is not needed.

File: api/Body.json [hunk 4]
```diff
@@ -0,0 +1,593 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "bodyUsed": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/bodyUsed",
```
Comment (Elchi3) [inline api/Body.json]:
```diff
-"https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/bodyUsed"
+"https://developer.mozilla.org/docs/Web/API/Body/bodyUsed"
```

File: api/Body.json [hunk 5]
```diff
@@ -0,0 +1,593 @@
...
+            "firefox": [
+              {
+                "version_added": "39"
+              },
+              {
+                "version_added": "34",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Body.json]: The name was "dom.fetch.enabled".

File: api/Body.json [hunk 6]
```diff
@@ -0,0 +1,593 @@
...
```
Comment (Elchi3) [inline api/Body.json]: The name was "dom.fetch.enabled".

File: api/Body.json [hunk 7]
```diff
@@ -0,0 +1,593 @@
...
```
Comment (Elchi3) [inline api/Body.json]: The name was "dom.fetch.enabled".

File: api/Body.json [hunk 8]
```diff
@@ -0,0 +1,593 @@
...
```
Comment (Elchi3) [inline api/Body.json]: The name was "dom.fetch.enabled".

File: api/Body.json [hunk 9]
```diff
@@ -0,0 +1,593 @@
...
```
Comment (Elchi3) [inline api/Body.json]: The name was "dom.fetch.enabled".

File: api/Body.json [hunk 10]
```diff
@@ -0,0 +1,593 @@
...
```
Comment (Elchi3) [inline api/Body.json]: The name was "dom.fetch.enabled".

File: api/Body.json [hunk 11]
```diff
@@ -0,0 +1,593 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Body.json]: "Experimental Web Platform Features"

File: api/Body.json [hunk 12]
```diff
@@ -0,0 +1,593 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Body.json]: "Experimental Web Platform Features"

File: api/Body.json [hunk 13]
```diff
@@ -0,0 +1,593 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Body.json]: "Experimental Web Platform Features"

File: api/Body.json [hunk 14]
```diff
@@ -0,0 +1,593 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Body.json]: "Experimental Web Platform Features"

File: api/Body.json [hunk 15]
```diff
@@ -0,0 +1,593 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Body.json]: "Experimental Web Platform Features"

File: api/Body.json [hunk 16]
```diff
@@ -0,0 +1,593 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Body.json]: "Experimental Web Platform Features"

File: api/Body.json [hunk 17]
```diff
@@ -0,0 +1,593 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Body.json]: "Experimental Web Platform Features"

File: api/Body.json [hunk 18]
```diff
@@ -0,0 +1,593 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Body.json]: "Experimental Web Platform Features"

File: api/Body.json [hunk 19]
```diff
@@ -0,0 +1,593 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Body.json]: "Experimental Web Platform Features"

File: api/Body.json [hunk 20]
```diff
@@ -0,0 +1,593 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Body.json]: "Experimental Web Platform Features"

File: api/Body.json [hunk 21]
```diff
@@ -0,0 +1,593 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "arrayBuffer": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/arrayBuffer",
```
Comment (Elchi3) [inline api/Body.json]:
```diff
-"https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/arrayBuffer"
+"https://developer.mozilla.org/docs/Web/API/Body/arrayBuffer"
```

File: api/Body.json [hunk 22]
```diff
@@ -0,0 +1,593 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "blob": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/blob",
```
Comment (Elchi3) [inline api/Body.json]:
```diff
-"https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/blob"
+"https://developer.mozilla.org/docs/Web/API/Body/blob"
```

File: api/Body.json [hunk 23]
```diff
@@ -0,0 +1,593 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "formData": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/formData",
```
Comment (Elchi3) [inline api/Body.json]:
```diff
-"https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/formData"
+"https://developer.mozilla.org/docs/Web/API/Body/formData"
```

File: api/Body.json [hunk 24]
```diff
@@ -0,0 +1,593 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "json": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/json",
```
Comment (Elchi3) [inline api/Body.json]:
```diff
-"https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/json"
+"https://developer.mozilla.org/docs/Web/API/Body/json"
```

File: api/Body.json [hunk 25]
```diff
@@ -0,0 +1,593 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "text": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/text",
```
Comment (Elchi3) [inline api/Body.json]:
```diff
-"https://developer.mozilla.org/docs/Web/API/TextMetrics/Body/text"
+"https://developer.mozilla.org/docs/Web/API/Body/text"
```

File: api/Body.json [hunk 26]
```diff
@@ -0,0 +1,593 @@
...
+  "api": {
+    "Body": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Body",
+        "support": {
+          "webview_android": {
+            "version_added": "42"
+          },
+          "chrome": {
+            "version_added": "42"
```
Comment (Elchi3) [inline api/Body.json]: This should likely also be:
```json
"chrome": [
              {
                "version_added": "42"
              },
              {
                "version_added": "41",
                "flags": [
                  {
                    "type": "preference",
                    "name": "Experimental Web Platform Features"
                  }
                ]
              }
            ],
```

Comment (maboa) [reply api/Body.json]: 👍

File: api/Body.json (hunk 1 above)
Comment (maboa) [reply api/Body.json]: 👍

**Comment by maboa** [comment]
Thanks @Elchi3 - hopefully all good now.

**Pull request merged**